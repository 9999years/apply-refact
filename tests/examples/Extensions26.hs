{-# LANGUAGE DeriveFunctor, GeneralizedNewtypeDeriving, StandaloneDeriving #-} 
deriving instance Show Bar