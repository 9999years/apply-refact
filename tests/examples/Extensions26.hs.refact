[("tests/examples/Extensions26.hs:1:1-78: Warning: Unused LANGUAGE pragma\nFound:\n  {-# LANGUAGE DeriveFunctor, GeneralizedNewtypeDeriving, StandaloneDeriving #-}\nPerhaps:\n  {-# LANGUAGE StandaloneDeriving #-}\nNote: Extension DeriveFunctor is not used, Extension GeneralizedNewtypeDeriving is not used\n",[ModifyComment {pos = SrcSpan {startLine = 1, startCol = 1, endLine = 1, endCol = 79}, newComment = "{-# LANGUAGE StandaloneDeriving #-}"}])]
