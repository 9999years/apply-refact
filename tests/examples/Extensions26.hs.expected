{-# LANGUAGE StandaloneDeriving #-}
deriving instance Show Bar
